plugins/aws/workspaces/workspacesDesiredBundleType.js

```diff
@@ -0,0 +1,53 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'WorkSpaces Desired Bundle Type',
+    category: 'WorkSpaces',
+    domain: 'Identity Access and Management',
+    description: 'Ensure that AWS WorkSpaces bundles are of desired types.',
+    more_info: 'A bundle in AWS WorkSpaces defines the hardware and software for AWS WorkSpaces. You can create a WorkSpaces instance using a predefined or custom bundle. Setting a limit to the types that can be used will help you control billing and address internal compliance requirements.',
+    recommended_action: 'Ensure that WorkSpaces instances are using desired bundle types',
+    link: 'https://docs.aws.amazon.com/workspaces/latest/adminguide/amazon-workspaces-bundles.html',
+    apis: ['WorkSpaces:describeWorkspaces'],
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+        var regions = helpers.regions(settings);
+
```

Comment:
Should have a setting for desired bundle types and compare each workspace against that type

Reply:
Added the settings for desired bundle type but we have to  check if the combined workspaces in a specific region have a single bundle type and if not this plugin will fail
As per the cloud conformity doc referenced here in the ticket https://scalock.atlassian.net/browse/SAAS-3110